src/eko/msbar_masses.py

```diff
@@ -332,16 +344,27 @@ def evolve(m2_ref, q2m_ref, strong_coupling, xif2, q2_to, nf_ref=None, nf_to=Non
     return m2_ref * ev_mass
 
 
-def compute(theory_card):
+def compute(
+    masses_ref: HeavyQuarkMasses,
+    couplings: CouplingsRef,
+    order: Order,
+    evmeth: CouplingEvolutionMethod,
+    matching: MatchingScales,
+    xif2: float = 1.0,
+):
     r"""Compute the |MSbar| masses.
 
     Computation is performed solving the equation :math:`m_{\bar{MS}}(\mu) =
     \mu` for each heavy quark and consistent boundary contitions.
 
     Parameters
     ----------
-    theory_card : dict
-        theory run card
+    nf_ref :
```

Comment:
this is out of sync

Reply:
I will fix. However, unfortunately, plenty of docstrings are at this point, so we will need a docstrings review campaign right before or right after merging